doc/tutorials/basic-minting-policies.rst

```diff
@@ -19,15 +19,18 @@ The minting policy is a function which receives these two inputs as *arguments*.
 The validating node is responsible for passing them in and running the minting policy.
 As with validator scripts, the arguments are passed encoded as :hsobj:`PlutusCore.Data.Data`.
 
-Using the script context
-------------------------
+Plutus V1 and V2 script context
+------------------------------------
 
 Minting policies have access to the :term:`script context` as their second argument.
-This will always be a value of type :hsobj:``PlutusLedgerApi.V1.Contexts.ScriptContext`` encoded as ``Data``.
+Plutus V1 and V2 minting policy scripts are differentiated only by their ``ScriptContext``. 
```

Comment:
Similar here, something like "Each version of Plutus minting policy scripts are ..."

Reply:
A minting policy script does not "have" a `ScriptContext`, so this reads a bit oddly. Perhaps "... are differentiated only by their `ScriptContext` argument"